src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -166,14 +93,14 @@ export default class Sect extends Module {
 		/*
 			SUGGESTION: Pulled without Sect
 		*/
-		if (!this.gaveup && (this.pullWithoutSect || !this.activeSectId)) {
+		if (!this.gaveup && (this.pullWithoutSect || this.activeSectId === 0)) {
 			this.suggestions.add(new Suggestion({
-				icon: !this.activeSectId || ACTIONS.DIURNAL_SECT.id === this.activeSectId ? ACTIONS.DIURNAL_SECT.icon : ACTIONS.NOCTURNAL_SECT.icon,
+				icon: this.activeSectId !== 0 || this.data.actions.DIURNAL_SECT.id === this.activeSectId ? this.data.actions.DIURNAL_SECT.icon : this.data.actions.NOCTURNAL_SECT.icon,
```

Comment:
I'd prob move this to a const so it's easier to read. Essentially the same check is done further down, so you can reuse the const.

Reply:
updated to be const for all three checks